tests/robustness/model/history.go

```diff
@@ -183,12 +183,16 @@ func toEtcdCondition(cmp clientv3.Cmp) (cond EtcdCondition) {
 	switch {
 	case cmp.Result == etcdserverpb.Compare_EQUAL && cmp.Target == etcdserverpb.Compare_MOD:
 		cond.Key = string(cmp.KeyBytes())
+		cond.ExpectedRevision = cmp.TargetUnion.(*etcdserverpb.Compare_ModRevision).ModRevision
 	case cmp.Result == etcdserverpb.Compare_EQUAL && cmp.Target == etcdserverpb.Compare_CREATE:
 		cond.Key = string(cmp.KeyBytes())
+		cond.ExpectedRevision = cmp.TargetUnion.(*etcdserverpb.Compare_ModRevision).ModRevision
```

Comment:
Don't know why we added parsing `Compare_CREATE` but we should not mix `CreateRevision` with `ModRevision`.

Reply:
Could you leave a TODO and/or send a followup PR for that?

Reply:
Removed. It doesn't seem to be used anywhere and it will be easy to add when needed